src/components/Modal/index.android.tsx

```diff
@@ -12,12 +12,13 @@ AppState.addEventListener('blur', () => {
     ComposerFocusManager.resetReadyToFocus();
 });
 
-// Only want to use useNativeDriver on Android. It has strange flashes issue on IOS
+// Only want to use useNativeDriver for backdrop on iOS. It has strange flashes issue on IOS
```

Comment:
Let's create a follow-up issue to investigate this and  potentially fix it?

Reply:
It's essentially this: https://github.com/Expensify/App/issues/18722. Is it better to create a new issue or simply just reopen that one?